### DISCUSSION TITLE
Nuxt SEO beta feedback

### DISCUSSION THREAD

**Discussion by harlan-zw** [harlan-zw/nuxt-seo#108] [General]
Nuxt SEO Kit v2 (renamed to Nuxt SEO) is in beta, providing:

- Hundreds of bug fixes that can't be ported to v1
- Runtime support (no prerendering required)
- Heaps of new features

See the [Installation guide](https://nuxtseo.com/nuxt-seo/getting-started/installation), [Migration guide](https://nuxtseo.com/nuxt-seo/getting-started/migration-guide) or jump in the <a href="https://discord.gg/275MBUBvgP">Discord</a> for support.

This discussion is for gathering feedback that will help shape the future of the SEO for Nuxt.

I need help with feedback on making sure the stable release of Nuxt SEO is a great DX. Please help me out with feedback on anything, some suggestions:
- Documentation that needs more work
- Undocumented features / breaking changes
- Missing features
- etc

Thanks!

**Comment by harlan-zw** [comment]
Related: https://github.com/harlan-zw/nuxt-seo-kit/discussions/116

**Comment by sosensible** [comment]
This page implies NuxtSEO is still not stable.
https://nuxtseo.com/nuxt-seo/getting-started/faq
